### ISSUE TITLE
Beacon crashing when attaching video to chat

### ISSUE THREAD

**Issue by NappyPirate** [helpscout/beacon-ios-sdk#100]
* Beacon version 2.0.1
* Beacon install method: [ ] Carthage, [X] CocoaPods, [ ] Manual

**Actual outcome**
On iOS, Beacon crashes the app when trying to attach a previously recorded video via the "Choose from Library" library option

**Expected outcome**
on iOS, Beacon should properly attach a previously recorded video via the "Choose from Library" option to an active chat without crashing the app

**Error Message**
```
Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Modifications to the layout engine must not be performed from a background thread after it has been accessed from the main thread.
```

This looks like it might be an issue with [iOS 13 specifically](https://stackoverflow.com/questions/58087536/modifications-to-the-layout-engine-must-not-be-performed-from-a-background-thr) and the use of UIKit on the keyboard thread instead of the main thread:
![Image 2020-09-02 at 1 54 23 PM](https://user-images.githubusercontent.com/6239901/92024550-f2b7ee00-ed23-11ea-8227-78fe3b292033.png)

**Comment by codenameDuchess** [comment]
Hi @NappyPirate, thanks for the bug report and sorry for the trouble. 
We'll look into this asap and let you know once we've got a fix in.

[Internal tracking ticket](https://helpscout.atlassian.net/browse/IOSSDK-332).

**Comment by ScottPetit** [comment]
Hey 👋 @NappyPirate ,

Thanks for reporting this, just wanted to write and let you know that this is resolved with the latest version (2.0.2) of the SDK.  If you'd like to see what else is in this release take a look at the [Changelog](https://github.com/helpscout/beacon-ios-sdk/blob/master/CHANGELOG.md).
